lib/gui/app/components/confirm-modal/ConfirmModal.jsx

```diff
@@ -16,19 +16,37 @@
 
 'use strict'
 
-/**
- * @module Etcher.Components.TargetSelector
- */
+const React = require('react')
+const { Modal } = require('rendition')
 
-const angular = require('angular')
-const { react2angular } = require('react2angular')
+const ConfirmModal = (options) => {
+  const {
+    confirmationLabel,
+    rejectionLabel,
+    title,
+    message,
+    cancel,
+    done
+  } = options
 
-const MODULE_NAME = 'Etcher.Components.TargetSelector'
-const SelectTargetButton = angular.module(MODULE_NAME, [])
+  const modalProps = {
```

Comment:
Let's apply this directly to the modal, instead of having it as a separate object.

Reply:
Deleted with the latest changes